introduction.md

```diff
@@ -6,4 +6,11 @@ Ex invenire temporibus est, hinc postea ullamcorper sea te. Agam habemus accusam
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?